packages/components/components/elvis-datepicker/src/react/calendar/calendar.tsx

```diff
@@ -80,8 +82,9 @@ export const Calendar: React.FC<Props> = ({
   };
 
   const formatCalendarDay = (date?: Date | null): string => {
-    const dateString = formatDate(date, { day: 'numeric' });
-    return dateString.substring(0, dateString.length - 1);
+    const dateString = formatDate(lang, date, { day: 'numeric' });
+    // console.info(dateString);
+    return dateString.replace('.', '');
```

Comment:
You might want to add a tiny comment explaining why we have to replace the period